composables/masto/account.ts

```diff
@@ -7,10 +7,14 @@ export function getDisplayName(account: mastodon.v1.Account, options?: { rich?:
   return displayName.replace(/:([\w-]+?):/g, '')
 }
 
+export function acctToShortHandle(acct: string) {
+  return `@${acct.includes('@') ? acct.split('@')[0] : acct}`
+}
+
 export function getShortHandle({ acct }: mastodon.v1.Account) {
   if (!acct)
     return ''
-  return `@${acct.includes('@') ? acct.split('@')[0] : acct}`
+  return toShortHandle(acct)
```

Comment:
Maybe it's `acctToShortHandle()` here?

Reply:
Good catch! Thanks for the review 🙏🏼 
Fixed at [`e32b483` (#1198)](https://github.com/elk-zone/elk/pull/1198/commits/e32b483a8afaa1da14258ce8d4d55d078de3fd8f)